compose/ui/ui-text/src/webCommonW3C/kotlin/androidx/compose/ui/text/intl/JsPlatformLocale.web.kt

```diff
@@ -54,12 +50,21 @@ private val rtlLanguagesSet = setOf("ar", "fa", "he", "iw", "ji", "ur", "yi")
 // since there is no js API for this.
 internal actual fun PlatformLocale.isRtl(): Boolean = this.language in rtlLanguagesSet
 
+// K/JS and K/Wasm stdlib doesn't have this type. Therefore, we declare it here.
+// Ideally it would not be necessary, or at least we would make it internal, but Compose common API
+// requires that expect PlatformLocale is actualized by a "standard/native" type of Locale.
 // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale
-internal external interface IntlLocale {
-    val language: String
-    val script: String?
-    val region: String?
-    val baseName: String
+// Note: Since Compose common code introduced PlatformLocale with extension properties with the same names,
+// we had to change the names of the properties in kotlin to avoid the name shadowing.
+external class IntlLocale {
+    @JsName("language")
+    val _language: String
```

Comment:
Can we make this `internal`?

Reply:
Yes. Done

Now it's more like a black box too. I like this.